src/main/kotlin/minesweeper/domain/Field.kt

```diff
@@ -1,64 +1,65 @@
 package minesweeper.domain
 
 class Field(
-    private val fieldInfo: FieldInfo,
-    private val mineCount: MineCount,
-    private val spotGenerator: SpotGenerator,
+    val fieldInfo: FieldInfo,
+    private val minePositions: Set<Position>,
 ) {
-    private val width = fieldInfo.getWidth()
-    val lines: List<FieldLine> = createField()
+    private val spots: Map<Position, Spot> = createField()
 
     init {
         validateMineCount()
     }
 
-    private fun createField(): List<FieldLine> {
-        val spots = spotGenerator.generate(fieldInfo, mineCount)
-        return spots.mapIndexed { index, spot ->
-            if (spot is SafeSpot) {
-                val y = index / width
-                val x = index % width
-                val nearbyMineCount = countAdjacentMines(spots, y, x)
-                spot.updateNearbyMineCount(nearbyMineCount)
+    private fun createField(): Map<Position, Spot> {
+        return (0 until fieldInfo.getWidth() + 1).flatMap { x ->
+            (0 until fieldInfo.getHeight() + 1).map { y ->
+                Position(x, y)
```

Comment:
.. 연산자를 활용해서 until 대신 사용해볼 수도 있겠어요!
```suggestion
        return (0..fieldInfo.getWidth()).flatMap { x ->
            (0..fieldInfo.getHeight()).map { y ->
                Position(x, y)
```